templates/registration/old/styling.css

```diff
@@ -0,0 +1,4 @@
+#map {
```

Comment:
Trust the versioning system [github] to maintain 'old' stuff for you - it's pretty easy to access and roll back to old code.

Reply:
If the 'old' folder's only purpose is to have a 'here's old versions' thing, you should delete i.t